node/get_price/src/index.js

```diff
@@ -0,0 +1,103 @@
+
+const getPrice= async function (req, res) {
+
+    const AlphavantageAPIkey = req.env.ALPHAVANTAGE_API_KEY;
+    const GoldAPIkey = req.env.GOLD_API_KEY; 
+    const CoinAPIkey = req.env.COIN_API_KEY;
+    const { type } = req.payload.toLowerCase();
+    let price,code;
+  
+    if (!type) {
+      throw new Error("type is required");
+    }
+  
+    if ((!AlphavantageAPIkey || AlphavantageAPIkey === "")&&(!GoldAPIkey || GoldAPIkey === "") && (!CoinAPIkey || CoinAPIkey === "")) {
```

Comment:
There's no need for both `!AlphavantageAPIkey` _AND_ `AlphavantageAPIkey === ""`. If `AlphavantageAPIkey` is an empty string, the first condition will be true aleady.